src/main/java/org/jenkinsci/plugins/workflow/FilePathUtils.java

```diff
@@ -89,6 +90,7 @@
      * @param path a path as returned by {@link FilePath#getRemote}
      * @return a corresponding file handle, if a node with that name is online, else null
      */
+    @SuppressFBWarnings(value="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification="TODO 1.653+ switch to Jenkins.getInstanceOrNull")
```

Comment:
@stephenc’s https://github.com/jenkinsci/jenkins/commit/bb7c8fcedbcc9b51c5b1bb5b32810af5ac6b1ffb causes annoyances like these when trying to make a single source base build across multiple baselines, and https://github.com/jenkinsci/jenkins/pull/2297 was actively incompatible.